.github/workflows/build-and-test-app.yml

```diff
@@ -0,0 +1,77 @@
+name: build-and-test
+run-name: build app from the ${{ github.head_ref }} branch
+on:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    - cron: "15 4,5 * * *"
+
+jobs:
+  build-app:
+    runs-on: macos-13
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup environment
+        env: 
+          XCODE_VERSION: 15.2
+        run: sudo xcode-select -s /Applications/Xcode_${{env.XCODE_VERSION}}.app
+
+      - name: Cache Swift dependencies
+        uses: actions/cache@v4
+        with:
+         path: ~/.swiftpm
+         key: ${{ runner.os }}-swift-${{ hashFiles('**/Package.resolved') }}
+
+      - name: Install xcpretty
+        run: |
+         gem install xcpretty
+         echo 'export PATH="$(gem environment gemdir)/bin:$PATH"' >> ~/.bash_profile
+
+      - name: Build iOS Simulator
+        run: ./scripts/build-ios-simulator.sh
+
+  test-woof-app:
+    runs-on: macos-13
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      
+      - name: Setup environment
+        env: 
+          XCODE_VERSION: 15.2
+        run: sudo xcode-select -s /Applications/Xcode_${{env.XCODE_VERSION}}.app
+
+      - name: Cache Swift dependencies
+        uses: actions/cache@v4
+        with:
+         path: ~/.swiftpm
+         key: ${{ runner.os }}-swift-${{ hashFiles('**/Package.resolved') }}
+
+      - name: Install xcpretty
+        run: |
+         gem install xcpretty
+         echo 'export PATH="$(gem environment gemdir)/bin:$PATH"' >> ~/.bash_profile
+
+      - name: Run tests
+        run: ./scripts/run-unit-tests.sh
+
+  run-linter-and-formater:
+    runs-on: macos-13
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        
+      - name: Run formater
+        continue-on-error: true
+        run: |
+         swiftformat --lint . \
+         --config .swiftformat \
+         --quiet
+
+      - name: Run linter
+        run: |
+         swiftlint --config .swiftlint.yml \
+         --strict \
+         --quiet
```

Comment:
Why do you prefer to run `formatter` and `linter` with --quiet flag?
Our code should not have warnings, but we will miss them.
WDYT?

Reply:
Warning are outputted, the `quiet`mode is concerned about the common output.